packages/framework-integration-tests/integration/providers/kubernetes/nuke/setup.ts

```diff
@@ -0,0 +1,23 @@
+import { overrideWithBoosterLocalDependencies } from '../../../helper/deps-helper'
+import { nuke } from '../deploy'
+import { createSandboxProject, removeSandboxProject } from '../../../../../cli/src/common/sandbox'
+import { sandboxPathFor } from '../../../helper/file-helper'
+import { sandboxProjectName } from '../constants'
+import { exec } from 'child-process-promise'
+import { createDaprComponentFile } from '../util'
+
+before(async () => {
+  const sandboxPath = sandboxPathFor(sandboxProjectName)
+  const configuredAssets = ['assets', 'assetFile.txt']
+  const sandboxedProject = createSandboxProject(sandboxPath, configuredAssets)
+
+  await overrideWithBoosterLocalDependencies(sandboxPath)
+  // Only the deploy command creates the production dependencies
+  await exec('npm install --production --no-bin-links', { cwd: sandboxPath })
+
+  // boost deploy removes the dapr components folder, so we need to create it again.
+  await createDaprComponentFile(sandboxPath)
```

Comment:
Just wondering: Could this be replaced by adding that file to the `config.assets` configuration field (and, thus, in the above `configuredAssets` variable)?

cc @rdiaz82

Reply:
oooh this worked hehe